src/api/getArticles.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
```

Comment:
Please create a base URL here with axios.create (see NC notes for Axios), also you can have all your functions in one file.